compiler/resolution/functionResolution.cpp

```diff
@@ -8809,17 +8790,13 @@ static void removeActualNames()
 
 static void removeTypeBlocks()
 {
-  forv_Vec(BlockStmt, block, gBlockStmts)
-  {
-    if (! block->parentSymbol)
-      continue;
-
+  for_alive_in_Vec(BlockStmt, block, gBlockStmts)
     // Remove type blocks--code that exists only to determine types
     if (block->blockTag & BLOCK_TYPE_ONLY)
     {
+      removeDeadTypeStuff(block);
```

Comment:
I think it would make sense to just inline the code in removeDeadTypeStuff here.

Reply:
I opted to keep removeDeadTypeStuff(), now removeRefAndAutoCopyForDeadTypes(), as it is quite distinct and also not that small, especially with the comments. Leaving it inside removeTypeBlocks() would divert the reader's attention, IMO.